plugins/kubernetes/lib/samson_kubernetes/samson_plugin.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'hash_kuber_selector'
+
+module SamsonKubernetes
+  class Engine < Rails::Engine
```

Comment:
I'm a fan of the [isolate_namespace](http://guides.rubyonrails.org/engines.html#critical-files) option. Especially since you can pretty neatly isolate the `Kubernetes` namespace from everything else.